stencil-library/src/components/business-details/representative-details/representative-details.tsx

```diff
@@ -44,7 +44,7 @@ export class JustifiRepresentativeDetails {
               />
               <DetailItem
                 title="Date Of Birth"
-                value={formatMediumDate(
+                value={this.representative && formatMediumDate(
```

Comment:
I added that, because I had the ErrorState removed in BusinessDetails, so I was trying to render the components without a representative loaded. I thought it would be better to just log the error on the console and Just show the component but without data. 
But thought better, and put back the ErrorState.